samples/app-hello-world/lib/export.py

```diff
@@ -0,0 +1,17 @@
+import lib.utils as utils
+import os
+import json
+
+try:
+    if "DATABOX_EXPORT_SERVICE_ENDPOINT" in os.environ:
+        exportServiceURL = os.environ['DATABOX_EXPORT_SERVICE_ENDPOINT']
+except NameError:
+        print("Export service endpoint is not defined")
+        exportServiceURL=''
+
```

Comment:
IIRC Python syntax correctly, this (L5-10 inclusive) can be simplified to something like
```
exportServiceURL = os.environ.get('DATABOX_EXPORT_SERVICE_ENDPOINT', '')
```